### PR TITLE
LaserShipV2 extra required parameters // umitguner

### PR DESCRIPTION
**Pull request by umitgnr**
# Description

LaserShipV2 extra required parameters added to the Options.cs class

# Testing

Extra three required parameter added for LaserShipV2 carrier.
The below documentation explains the why these parameters needed and test conditions and results.
https://support.easypost.com/hc/en-us/articles/360062036932-Lasership-V2-Guide

# Pull Request Type

- [ ] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Comment by umitgnr** [comment]
the pull request related to easypost support ticket number "321557"

**Comment by jchen293** [comment]
@umitgnr Thanks for the PR! You will need to remove the `pickup_min_datetime` since there is already one in line 133

**Comment by Justintime50** [comment]
Please also place the other options into the listed sorted instead of at the end of the list

**Review by umitgnr — COMMENTED** [review]
LaserShipV2 changes fixed and sorted // umitguner

**Comment by umitgnr** [comment]
Thank you for feedback, it is updated and sorted.

Regards,

Justin Hammond ***@***.***>, 22 Eyl 2022 Per, 16:59 tarihinde
şunu yazdı:

**Review by nwithan8 — APPROVED** [review]

File: EasyPost/Options.cs [hunk 1]
```diff
@@ -180,7 +184,6 @@ public class Options : Resource
         public string special_rates_eligibility { get; set; }
         [JsonProperty("suppress_etd")]
         public bool? suppress_etd { get; set; }
-
```
Comment (jchen293) [inline EasyPost/Options.cs:183]: This line should be reserved

**Review by jchen293 — APPROVED** [review]

**Review by Justintime50 — APPROVED** [review]
LaserShip definitely requires these fields. Thanks for adding.

**Pull request merged**